webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -375,58 +92,27 @@ const WorkspaceDetails: React.FC = () => {
         />
       )}
       id="workspaceDetails">
-      <div className={css.controls}>
-        <SelectFilter
-          dropdownMatchSelectWidth={140}
-          showSearch={false}
-          value={settings.whose}
-          onSelect={handleViewSelect}>
-          <Option value={WhoseProjects.All}>All Projects</Option>
-          <Option value={WhoseProjects.Mine}>My Projects</Option>
-          <Option value={WhoseProjects.Others}>Others&apos; Projects</Option>
-        </SelectFilter>
-        <Space wrap>
-          {!workspace.archived && (
-            <Toggle
-              checked={settings.archived}
-              prefixLabel="Show Archived"
-              onChange={switchShowArchived}
-            />
-          )}
-          <SelectFilter
-            dropdownMatchSelectWidth={150}
-            showSearch={false}
-            value={settings.sortKey}
-            onSelect={handleSortSelect}>
-            <Option value={V1GetWorkspaceProjectsRequestSortBy.NAME}>Alphabetical</Option>
-            <Option value={V1GetWorkspaceProjectsRequestSortBy.LASTEXPERIMENTSTARTTIME}>
-              Last Updated
-            </Option>
-            <Option value={V1GetWorkspaceProjectsRequestSortBy.CREATIONTIME}>
-              Newest to Oldest
-            </Option>
-          </SelectFilter>
-          <GridListRadioGroup value={settings.view} onChange={handleViewChange} />
-        </Space>
-      </div>
-      <Spinner spinning={isLoading}>
-        {projects.length !== 0 ? (
-          projectsList
-        ) : (
-          workspace.numProjects === 0 ? (
-            <Message
-              message='Create a project with the "New Project" button or in the CLI.'
-              title="Workspace contains no projects. "
-              type={MessageType.Empty}
-            />
-          ) : (
-            <Message
-              title="No projects matching the current filters"
-              type={MessageType.Empty}
-            />
-          )
-        )}
-      </Spinner>
+      {
+        rbacEnabled ? (
+          <Tabs
+            activeKey={tabKey}
+            destroyInactiveTabPane
+            onChange={handleTabChange}>
+            <Tabs.TabPane
+              destroyInactiveTabPane
+              key={WorkspaceDetailsTab.Projects}
+              tab="Projects">
+              <WorkspaceProjects id={id} pageRef={pageRef} workspace={workspace} />
+            </Tabs.TabPane>
+            <Tabs.TabPane
```

Comment:
I recommend checking permission like PERMISSION_CAN_UPDATE_WORKSPACE to display `Members` tab